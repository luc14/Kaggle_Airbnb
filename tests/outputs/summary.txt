	learner	acc_train_mean	acc_train_std	acc_val_mean	acc_val_std	logloss_train_mean	logloss_train_std	logloss_val_mean	logloss_val_std	ndcg_train_mean	ndcg_train_std	ndcg_val_mean	ndcg_val_std	age_na	id	info_str	input_folder	join_sessions	learners	output_folder	scale	session	submission	base_score	colsample_bylevel	colsample_bytree	gamma	learning_rate	max_delta_step	max_depth	min_child_weight	missing	n_estimators	nthread	objective	reg_alpha	reg_lambda	scale_pos_weight	seed	silent	subsample	version	training rows	training columns
0	XGBClassifier	0.8362068965517241		0.6060606060606061		-0.49991242341622283		-1.2847705442091506		0.932400375389987		0.8018825989360217				tests/info_str.txt	tests/	outer	['xgb']	tests/outputs/	False	True	True	0.5	1	0.5	0	0.3	0	6	1		25	-1	binary:logistic	0	1	1	0	True	0.5	1.0	414	271

