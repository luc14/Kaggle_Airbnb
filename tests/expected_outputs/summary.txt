	learner	acc_train_mean	acc_train_std	acc_val_mean	acc_val_std	logloss_train_mean	logloss_train_std	logloss_val_mean	logloss_val_std	ndcg_train_mean	ndcg_train_std	ndcg_val_mean	ndcg_val_std	dummy	logreg	nn	scale	session	small	submission	test	xgb	base_score	colsample_bylevel	colsample_bytree	gamma	learning_rate	max_delta_step	max_depth	min_child_weight	missing	n_estimators	nthread	objective	reg_alpha	reg_lambda	scale_pos_weight	seed	silent	subsample	switched from intersection to union
0	XGBClassifier	0.7710914454277286		0.7028985507246377		-0.5517996981267469		-0.7893097922070951		0.9103685091135841		0.87992928193758		False	False	False	False	True	True	True	True	True	0.5	1	0.5	0	0.3	0	6	1		25	-1	binary:logistic	0	1	1	0	True	0.5	True

